src/main/java/lotto/enums/Rank.java

```diff
@@ -0,0 +1,40 @@
+package lotto.enums;
+
+public enum Rank {
+    ZERO(0, false, 0, " "),
+    ONE(6, false, 2000000000, "6개 일치 (2,000,000,000원)"),
```

Comment:
언더바를 이용하면 숫자의 가독성을 더 좋게 만들 수 있습니다!
```suggestion
    ONE(6, false, 2_000_000_000, "6개 일치 (2,000,000,000원)"),
```

Reply:
확실히 가독성이 더 좋아졌네요 !!
피드백 감사합니다 👍